docs/events/com.mbta.rtr.lightrail-vehicle-information.mdx

```diff
@@ -0,0 +1,22 @@
+---
+---
+
+import LightRailVehicleInformation from "@site/examples/com.mbta.rtr.light-vehicle-information/LightRailVehicleInformation.json";
+import EventSchemaPath from "@site/src/components/EventSchemaPath.tsx";
+import Examples from "@site/src/components/Examples.tsx";
+
+Message from RTR indicating information about the location and status of light rail vehicles provided by RTR. RTR sources this information from the OCS and/or the vehicle AVLs.
```

Comment:
Would RTR ever delete a vehicle, or just let it go stale? If it is deleted, how?

Reply:
Currently, Glides handles it by completely replacing our list of vehicles every time we poll. But with this event stream sending updates one vehicle at a time, we wouldn't be able to do that, so there'd need to be an explicit way to update a vehicle to be deleted.

Reply:
would glides be able to discard updates for vehicles that aren't in its list of vehicles? Otherwise, we could  send another notification with something like `isDeleted` when we get a TmovDel

Reply:
Could deletions be in another key at the same level as `data`, or does that violate CloudEvents? I don't love `isDeleted`, as it would contain a lot of extraneous keys (and superfluous, old data?)?

Reply:
> yes, if OCS sends us a lat/lon of 0.0/0.0 for a train then we delete it. You can [see this happening in prod here](https://mbta.splunkcloud.com/en-US/app/search/search?q=search%20index%3D%22rtr-prod%22%20%22delete%20msg%20for%20train%22&display.page.search.mode=fast&dispatch.sample_ratio=1&workload_pool=&earliest=-30m&latest=now&sid=1741879088.60544). Would you want some mechanism by which RTR informs Glides that the vehicle is being deleted, or is that being handled elsewhere?


For completion, this isn't actually how it works for LRVs. These are cleaned up by the `remove_stale_vehicles` function which deletes any vehicles that haven't received any updates for the past 30 minutes. You can search for logs with `deleted_stale_vehicle` from RTR.

Reply:
@mathcolo  there is no context attribute in the CloudEvents schema for denoting a deletion, so you would have to shoehorn it into data.

IME the deletion of a train is separate from this information event. how about a separate event for saying that a vehicle has been deleted?

Reply:
I added a separate event for deletion that only specifies `vehicleId`. let me know what you think